config/migrations/2025/20251222142112-restore-data-fixes/20251211105414-fix-ovo.sparql

```diff
@@ -0,0 +1,146 @@
+
+  }
+}
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/organizations/780e50a94a76c850a8f88c7e01427de38b445dcc6ebe79815a6d0f44adc530bc/LoketLB-LPDCGebruiker> {
+<http://data.lblod.info/id/public-service/bcbe9f96-6390-4c67-9e61-f2d774fbf153>	<https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#hasExecutingAuthority>	<http://data.lblod.info/id/bestuurseenheden/780e50a94a76c850a8f88c7e01427de38b445dcc6ebe79815a6d0f44adc530bc> .
+<http://data.lblod.info/id/public-service/bcbe9f96-6390-4c67-9e61-f2d774fbf153>	<http://data.europa.eu/m8g/hasCompetentAuthority>	<http://data.lblod.info/id/bestuurseenheden/4f4a3d2d9a87bad57bdb770b0c52b1fe6fe4064fb48f1279496f35cd3fe3c1f5> .
+<http://data.lblod.info/id/public-service/bcbe9f96-6390-4c67-9e61-f2d774fbf153>	<http://purl.org/dc/terms/spatial> <http://data.europa.eu/nuts/code/BE24224008> .
+  }
+}
+
+## Lennik
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/organizations/931a2bf4f5c461d8c3636b446fa642693d4535d865d3af068e87d25ddc93374e/LoketLB-LPDCGebruiker> {
+<http://data.lblod.info/id/public-service/8371c9e5-fd79-417a-93e0-487ab8eb41ba>	<http://data.europa.eu/m8g/hasCompetentAuthority>	<https://data.vlaanderen.be/id/organisatie/OVO002147> .
+<http://data.lblod.info/id/public-service/8371c9e5-fd79-417a-93e0-487ab8eb41ba>	<https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#hasExecutingAuthority>	<https://data.vlaanderen.be/id/organisatie/OVO002147> .
+<http://data.lblod.info/id/public-service/8371c9e5-fd79-417a-93e0-487ab8eb41ba>	<http://data.europa.eu/m8g/hasCompetentAuthority>	<https://data.vlaanderen.be/id/organisatie/OVO002505> .
+
+<http://data.lblod.info/id/public-service/4c6409f0-0a68-45e8-805c-7319d8b0bdcf>	<https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#hasExecutingAuthority>	<https://data.vlaanderen.be/id/organisatie/OVO002147> .
+<http://data.lblod.info/id/public-service/4c6409f0-0a68-45e8-805c-7319d8b0bdcf>	<http://data.europa.eu/m8g/hasCompetentAuthority>	<https://data.vlaanderen.be/id/organisatie/OVO002505> .
+
+<http://data.lblod.info/id/public-service/d9f97aeb-8e28-4842-8ee8-774d606cc7bd>	<https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#hasExecutingAuthority>	<https://data.vlaanderen.be/id/organisatie/OVO002147> .
+<http://data.lblod.info/id/public-service/d9f97aeb-8e28-4842-8ee8-774d606cc7bd>	<http://data.europa.eu/m8g/hasCompetentAuthority>	<https://data.vlaanderen.be/id/organisatie/OVO002505> .
+
+  }
+}
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/organizations/931a2bf4f5c461d8c3636b446fa642693d4535d865d3af068e87d25ddc93374e/LoketLB-LPDCGebruiker> {
+<http://data.lblod.info/id/public-service/8371c9e5-fd79-417a-93e0-487ab8eb41ba>	<http://data.europa.eu/m8g/hasCompetentAuthority>	<http://data.lblod.info/id/bestuurseenheden/931a2bf4f5c461d8c3636b446fa642693d4535d865d3af068e87d25ddc93374e> .
```

Comment:
Checking `Babyspullen ontlenen in de Babytheek` i found that Lint OCMW has been added too. 
That should be removed right? 
http://localhost/8371c9e5-fd79-417a-93e0-487ab8eb41ba/eigenschappen